Jellyfin.Plugin.SmartPlaylist/Configuration/config.js

```
@@ -1008,73 +1079,403 @@
+        return formatLegacySchedule(playlist);
+    }
+    
+    function formatSingleSchedule(schedule) {
+        if (schedule.Trigger === 'Daily') {
+            var raw = schedule.Time ? schedule.Time.substring(0, 5) : '00:00';
+            var parts = raw.split(':');
+            var h = parts[0] !== undefined ? parseInt(parts[0], 10) : 0;
+            var m = parts[1] !== undefined ? parseInt(parts[1], 10) : 0;
+            return 'Daily at ' + formatTimeForUser(h, m);
+        } else if (schedule.Trigger === 'Weekly') {
+            var raw = schedule.Time ? schedule.Time.substring(0, 5) : '00:00';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'raw'. Consider to delete it or rename it.

'raw' is defined here:


(lint/suspicious/noRedeclare)